subscriptions/templatetags/seed.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+import seed_stage_based_messaging
+
+register = template.Library()
+
+
+@register.simple_tag
+def current_version():
+    return seed_stage_based_messaging.__version__
```

Comment:
This is probably one of the benefits of using __init__.py for storing the version number.. 🤔